apps/backend/src/users/users.controller.ts

```diff
@@ -27,4 +27,10 @@ export class UsersController {
   removeUser(@Param('id') id: string) {
     return this.usersService.remove(parseInt(id));
   }
+
+  @UseGuards(AuthGuard('jwt'))
+  @Get('/pantries/:pantryId/ssf-contact')
```

Comment:
Maybe it would make more sense to put this route under a pantries controller so that the path is just /pantries/{pantryId}/ssf-contact instead of with the users/ prefix? I know that this is getting a user in the end but the representative is kind of like an attribute of a specific pantry that we query for